changelogs/fragments/5224-proxmox-unprivileged-default.yaml

```
@@ -0,0 +1,2 @@
+deprecated_features:
+  - proxmox module - Deprecated the current ``unprivileged`` default value, will be changed to ``true`` in 7.0.0. (https://github.com/ansible-collections/community.general/pull/5224)
```

We punctuate the change logs.
```suggestion
  - proxmox module - Deprecated the current ``unprivileged`` default value, will be changed to ``true`` in 7.0.0. (https://github.com/ansible-collections/community.general/pull/5224).
```
They are later collected and become something like this one:
https://github.com/ansible-collections/community.general/blob/stable-5/CHANGELOG.rst#release-summary